pom.xml

```diff
@@ -44,7 +44,7 @@
     <dropwizard.json.log.bundle.version>1.0.2-0</dropwizard.json.log.bundle.version>
     <dropwizard.metrics.datadog.ecs.version>0.4.0</dropwizard.metrics.datadog.ecs.version>
```

Comment:
We can consolidate all of the java-commons libraries to one version now. This is something we updated for 1.1.4: https://github.com/mainstreethub/java-commons

Reply:
So should I include java-commons as a dependency and remove the instances of those already included dependencies from the dropwizard-parent-pom?

Reply:
So should I include java-commons as a dependency and remove the instances of those already included dependencies from the dropwizard-parent-pom?

Reply:
We don't have a BOM right now for java-commons, so that is an option to add one. But the simplest thing to do here would be to specify a property, `<java.commons.version>1.1.4-1</java.commons.version>`, and specify it for each of the java-commons dependencies.